config.go

```diff
@@ -174,8 +173,11 @@ func loadConfig() (*config, []string, error) {
 		return nil, nil, err
 	}
 
-	// Update LndNode using the activeNetParams
-	cfg.LndNode = "localhost:" + activeNetParams.rpcPort
+	// If user don't sets the LndNode, then update LndNode
+	// using the activeNetParams to deafult (localhost:<default_port>).
```

Comment:
"If user didn't set the LndNode to connect to then use the default port for the active network (localhost:<default_port>)."